package.json

```diff
@@ -69,7 +63,7 @@
     "commitlint-plugin-function-rules": "2.0.2",
     "concurrently": "8.2.1",
     "cross-env": "7.0.3",
-    "husky": "8.0.3",
+    "husky": "9.0.11",
```

Comment:
I think we should be safe to allow non-breaking semver upgrades here as husky follows semantic versioning. 

By adding `^` to the version e.g. `"husky": "^9.0.11"`, this will mean that the minor versions can be upgraded when the lock files are updated.